### PR TITLE
Isolation between multiple teams on a single instance

### PR DESCRIPTION
**Pull request by ronnix**

### CODE REVIEW

File: repondeur/zam_repondeur/models/base.py [hunk 1]
```diff
@@ -31,7 +31,12 @@ def __repr__(self) -> str:
         extension=ZopeTransactionExtension(),  # attach to the transaction manager
     )
 )
-metadata = MetaData()
+
+metadata = MetaData(
+    # Use an explicit naming convention to help Alembic autogenerate
```
Comment (davidbgk) [inline repondeur/zam_repondeur/models/base.py]: 👏

File: repondeur/zam_repondeur/__init__.py [hunk 2]
```diff
@@ -131,10 +134,26 @@ def get_user(request: Request) -> Optional[User]:
...
     return None
 
 
+def get_team(request: Request) -> Optional[Team]:
+    team_name = request.environ.get("REMOTE_USER")
+    if team_name is None:
+        return None
+    team: Optional[Team] = DBSession.query(Team).filter_by(name=team_name).first()
+    if team is None:
+        team = Team(name=team_name)
```
Comment (davidbgk) [inline repondeur/zam_repondeur/__init__.py]: Is that intentional not using the `.create()` pattern?

File: repondeur/zam_repondeur/resources.py [hunk 3]
```diff
@@ -16,6 +16,9 @@
 )
 
 
+ACE = Tuple[str, str, str]
```
Comment (davidbgk) [inline repondeur/zam_repondeur/resources.py]: Access Control… Entry?

Comment (ronnix) [reply repondeur/zam_repondeur/resources.py]: Yes: https://docs.pylonsproject.org/projects/pyramid/en/latest/glossary.html#term-ace

File: repondeur/zam_repondeur/__init__.py (hunk 2 above)
Comment (ronnix) [reply repondeur/zam_repondeur/__init__.py]: Nah, we should be consistent and use create here too.

**Comment by ronnix** [comment]
Je m’en occupe

File: repondeur/zam_repondeur/templates/login.html [hunk 4]
```diff
@@ -8,8 +8,11 @@ <h1>Entrer dans Zam</h1>
         {% if request.session.pop('missing_email', None) %}
             <p class="error">La saisie d’une adresse de courriel est requise.</p>
         {% endif %}
+        {% if request.session.pop('already_in_use', None) %}
+            <p class="error">Cette addresse de courriel est déjà utilisée<br>dans une autre équipe.</p>
```
Comment (davidbgk) [inline repondeur/zam_repondeur/templates/login.html]: `adresse` ;)

File: repondeur/zam_repondeur/templates/login.html [hunk 5]
```diff
@@ -8,8 +8,11 @@ <h1>Entrer dans Zam</h1>
...
+        {% endif %}
         <label for="email">Votre courriel</label>
-        <input type="email" name="email" autocomplete="email" required autocorrect="off" spellcheck="false">
```
Comment (davidbgk) [inline repondeur/zam_repondeur/templates/login.html]: Why not anymore required?

File: repondeur/zam_repondeur/templates/login.html (hunk 4 above)
Comment (ronnix) [reply repondeur/zam_repondeur/templates/login.html]: Bien vu, merci !

File: repondeur/zam_repondeur/templates/login.html (hunk 5 above)
Comment (ronnix) [reply repondeur/zam_repondeur/templates/login.html]: Just a test, should not have been committed. Thanks for catching it!

**Pull request merged**